recipes/gunicorn/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "gunicorn" %}
+{% set version = "19.6.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version}}.tar.gz
+  url: https://github.com/benoitc/gunicorn/archive/{{ version }}.tar.gz
+  sha256: 1e0de4957bea60bfcff5215664bdfc3cf02c78e2aae9586766a9b4b437aebbb0
+  patches:
+    # See https://github.com/conda-forge/staged-recipes/pull/885#discussion_r68602712
+    - no_gaiohttp.patch  # [py2k]
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  preserve_egg_dir: yes
+  entry_points:
+    - gunicorn=gunicorn.app.wsgiapp:run
+    - gunicorn_django=gunicorn.app.djangoapp:run
+    - gunicorn_paster=gunicorn.app.pasterapp:run
+
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+    - gunicorn
+    - gunicorn.app
+    - gunicorn.http
+    - gunicorn.instrument
+    - gunicorn.management
+    - gunicorn.management.commands
+    - gunicorn.workers
+  commands:
+    - gunicorn --help
+    - gunicorn_django --help
+    - gunicorn_paster --help
+  requires:
+    - pytest
+    - pytest-cov
+    - paste
```

Comment:
Please add in another PR b/c after `gunicorn` and `paste` will start at the same time forcing you to re-start the feedstock after the merge since the `paste` might not be ready yet.